src/sentry/features/temporary.py

```diff
@@ -47,6 +47,7 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:anr-improvements", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:api-auth-provider", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:api-keys", OrganizationFeature, FeatureHandlerStrategy.INTERNAL)
+    manager.add("organizations:api-organization_events-rate-limit-reduced-rollout", OrganizationFeature, FeatureHandlerStrategy.OPTIONS)
```

Comment:
is `OPTIONS` handler strategy ready to roll? have we tested it on a less high stakes option in prod?

Reply:
@wedamija how confident do you feel about this?

Reply:
going back to the old school way for now to minimize the risk